manager/orchestrator/restart/restart.go

```diff
@@ -190,6 +197,83 @@ func (r *Supervisor) Restart(ctx context.Context, tx store.Tx, cluster *api.Clus
 	return nil
 }
 
+// TaskRestartDelay calculates and returns the next delay based t's RestartPolicy.
+// It also returns whether the caller should randomized the value.
+// Important: It is the caller's responsibility to randomize the returned duration.
+func (r *Supervisor) TaskRestartDelay(ctx context.Context, t *api.Task) (time.Duration, bool, error) {
```

Comment:
why not use 's' for supervisor instead of 'r' ?

Reply:
We were following the convention of the rest of the file.